docs/guides/latest/volumesnapshot/deployment-backup.md

```diff
@@ -0,0 +1,262 @@
+## Backup Deployment's Volumes
+
+This tutorial will show you how to use Stash Operator to backup Deployment's volumes using [Volume Snapshot](https://kubernetes.io/docs/concepts/storage/volume-snapshots/) API resources.
+
+##### Before You Begin
+
+* At first, you need to familiarize with the GCE CSI-driver, to enable the Volume Snapshotting feature via feature gate and to install Stah Operator in your cluster. If you are not familiar with that please visit [here](docs/guides/latest/volumesnapshot/backup.md) 
+
+To keep everything isolated, we are going to use a separate namespace called `demo` throughout this tutorial.
+
+```console
+$ kubectl create ns demo
+namespace/demo created
+```
+
+Now, we will create a StorageClass for further uses. 
+Sample StorageClass YAML are given below,
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: standard
+parameters:
+  type: pd-standard
+provisioner: pd.csi.storage.gke.io
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+```
+The [volumeBindingMode](https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode) field controls when volume binding and dynamic provisioning should occur. By default, the `Immediate` mode indicates that volume binding and dynamic provisioning occurs once the PVC is created and `WaitForFirstConsumer` mode indicates that volume binding and provisioning occurs until a pod using the PVC is created.
```

Comment:
Remove `By default,` from `By default, the Immediate`

Reply:
`provisioning occurs until a pod using the PVC is created.` -> `provisioning does not occur until a pod using the PVC is created.`